### PR TITLE
Create Readme.md Dijkstra Algo

### LINKED ISSUE #2204: Dijkstra Algorithm
- [x] C
- [x] C++
- [ ] C# - @anshu18107
- [ ] CoffeeScript
- [x] Dart - @Swarnimashukla
- [ ] Go - @raksha009
- [x] Java
- [ ] JS - @abhu-A-J
- [ ] Kotlin - @garimasingh128
- [x] Php - @sukritishah15
- [x] Python
- [ ] Readme - @shwetagurnani
- [x] Ruby
- [ ] Rust - @kichloo
- [x] TypeScript

### PR DESCRIPTION
**Pull request by shwetagurnani**
Fixes #2204

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [ ] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Created readme for dijkstra Algorithm
- 

#### Languages Used:
- 
- 

#### Files Added:
- readme.md
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by shwetagurnani** [comment]
@jainaman224 Please review this pull request.Its more than a week I have made this PR!

File: Dijkstra_Algorithm/Readme.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+                if(not(sptSet[v]) && graph[u][v] && dist[u] not equal to INT_MAX && dist[u] + graph[u][v] < dist[v]) 
+                     dist[v] = dist[u] + graph[u][v]
+        for(i = 0; i < V; i++)
+            display i, dist[i]
+```
+
+**Using C++ STL**
+
+```  
+
```
Comment (asha15) [inline Dijkstra_Algorithm/Readme.md]: remove this line

File: Dijkstra_Algorithm/Readme.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+               delete s.begin
+               u = temp.second
+               for(i = adj[u].begin(); i != adj[u].end(); i++)
+                   v = (*i).first
+                   weight = (*i).second
+                   if dist[v] != INF
+                       delete(dist[v], v)
+                   s.insert(dist[v], v)
+           for(i = 0; i < V; i++)
+               display i, dist[i]
```
Comment (asha15) [inline Dijkstra_Algorithm/Readme.md]: keep a proper indentation

File: Dijkstra_Algorithm/Readme.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+```
+       
+## Complexity
+       Time Complexity: O(V * V)
+       Space Complexity: O(V)
```
Comment (asha15) [inline Dijkstra_Algorithm/Readme.md]: remove the extra spaces

File: Dijkstra_Algorithm/Readme.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+       
+ **Using C++ STL:**
+ 
+       Time Complexity: O(V + ElogV)
+       Space Complexity: O(V)
```
Comment (asha15) [inline Dijkstra_Algorithm/Readme.md]: remove the extra spaces

**Comment by somya-kapoor** [comment]
Due to no activity for more than 15 days, I am closing this PR.

**Pull request closed without merging**